src/test/java/com/keeper/homepage/domain/election/application/AdminElectionServiceTest.java

```diff
@@ -52,5 +68,138 @@ class DeleteElection {
+
+      List<Long> candidateIds = IntStream.range(0, 5)
+          .mapToObj(member -> memberTestHelper.generate().getId())
+          .collect(toList());
+
+      electionCandidateRepository.deleteAll();
+      adminElectionService.registerCandidates(candidateIds, electionCandidate.getDescription(), memberJob.getId(),
+          electionId);
+
+      em.flush();
+      em.clear();
+
+      List<ElectionCandidate> savedCandidate = electionCandidateRepository.findAll();
+
+      assertEquals(5, savedCandidate.size());
+    }
+
+    @Test
+    @DisplayName("후보자 다중 등록 null check")
+    public void 후보자_다중_등록_null_check() throws Exception {
+      List<Long> candidateIds = IntStream.range(0, 5)
+          .mapToObj(member -> (member == 2) ? null : memberTestHelper.generate().getId())
+          .collect(toList());
+
+      ElectionCandidatesRegisterRequest request = ElectionCandidatesRegisterRequest.builder()
+          .candidateIds(candidateIds)
+          .description("후보")
+          .memberJobId(memberJobId)
+          .build();
+
+      Set<ConstraintViolation<ElectionCandidatesRegisterRequest>> violations = validator.validate(request);
+
+      assertEquals(1, violations.size());
+
+      /*
+          결과 설명 추가
+          1. @NotEmpty(message = "후보자 아이디를 입력해주세요.")
+             private List<@NotNull Long> candidateIds;     => 성공
+          2. @NotEmpty(message = "후보자 아이디를 입력해주세요.")
+             private List<Long> candidateIds;              => 실패
+          3. private List<@NotNull Long> candidateIds;     => 성공
+          4. @NotNull(message = "후보자 아이디를 입력해주세요.")
+             private List<Long> candidateIds;              => 실패
+      */
+    }
+
+    @Test
+    @DisplayName("후보자 삭제에 성공한다.")
+    public void 후보자_삭제에_성공한다() throws Exception {
+      long electionId = electionCandidate.getElection().getId();
+      long candidateId = electionCandidate.getId();
+      adminElectionService.deleteCandidate(electionId, candidateId);
+
+      em.flush();
+      em.clear();
+
+      assertThat(electionCandidateRepository.findById(candidateId)).isEmpty();
+    }
+
+    @Test
+    @DisplayName("후보자 정보가 없다면 후보자 삭제에 실패한다.")
+    public void 후보자_정보가_없다면_후보자_삭제에_실패한다() throws Exception {
+      long electionId = electionCandidate.getElection().getId();
+      long notExistCandidateId = 10;
+
+      assertThrows(BusinessException.class, () -> adminElectionService.deleteCandidate(electionId, notExistCandidateId),
+          "해당 후보자를 찾을 수 없습니다.");
+    }
+
+    @Test
+    @DisplayName("선거가 공개 상태라면 후보자 삭제는 실패한다.")
+    public void 선거가_공개_상태라면_후보자_삭제는_실패한다() throws Exception {
+      Election election = electionTestHelper.builder()
+          .name("제 1회 임원진 선거")
+          .description("임원진 선거입니다.")
+          .isAvailable(true)
+          .build();
+
+      assertThrows(BusinessException.class, () -> adminElectionService.deleteCandidate(election.getId(),
+          electionCandidate.getId()), "비공개 상태 선거에서만 후보자 삭제가 가능합니다.");
```

Comment:
```suggestion
          electionCandidate.getId()), ELECTION_CANDIDATE_CANNOT_DELETE.getMessage());
```

이렇게 할 수 있습니다! 이렇게 하면 메세지가 변경되어도 테스트를 수정하지 않아도 될 것 같아요~!